site/content/contribute/good-decisions/fixing-pr-mistakes.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Merged PR Scenario"
+heading: "Merged PR Scenario"
+weight: 2
+---
+
+This document's aim is to explain what process should be in place in the following scenario:
```

Comment:
Let's introduce the scenario less formally as something along the lines of "This page describes the process to follow when someone notices a mistake in a merged pull request (PR)."

Reply:
Done.